test/functional/fixtures/runner/using-shared-code/cross-domain.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>

```

Comment:
there is inconsistency in file naming.
`cross-domain.html`, `same-domain.html`, but `inline-iframe.html`.
We should use the `-iframe` postfix in all files or in nowhere. I prefer the second way